tfhe/src/zk.rs

```diff
@@ -16,6 +18,14 @@ impl Named for CompactPkeProof {
     const NAME: &'static str = "zk::CompactPkeProof";
 }
 
+impl ParameterSetConformant for CompactPkeProof {
+    type ParameterSet = ();
+
+    fn is_conformant(&self, _parameter_set: &Self::ParameterSet) -> bool {
+        self.is_usable()
+    }
+}
+
```

Comment:
Shouldn't `is_usable` also be checked on `proof` in the loop in `impl ParameterSetConformant for ProvenCompactCiphertextList`